retrieve_09S2y000000iMwPEAU/classes/OpportunityController.cls

```diff
@@ -0,0 +1,40 @@
+public class OpportunityController 
+{
+    public Opportunity myOpportunity{get; set;}
+    public List<Contact> contactList{get; set;}
+    public Id contactId{set; get;}
+    
+    public OpportunityController()
+    {
+   		Id opportunityId =  ApexPages.currentPage().getParameters().get('Id');
+    	myOpportunity = [SELECT Id, Name, BillToContact__c, Manager__c FROM Opportunity WHERE Id =: opportunityId];     
+    	contactList = [SELECT Id, Name FROM Contact WHERE Account.Id =: myOpportunity.Manager__c];
```

Comment:
`Account` is the relationship name for account on `Contact` the reference field could also be used as `AccountId`